examples/smart-mastering-complete/src/main/entity-config/exp-final-entity-options.xml

```diff
@@ -3,6 +3,8 @@
   <search:constraint name="Collection">
     <search:collection>
       <search:facet-option>limit=25</search:facet-option>
+      <search:facet-option>frequency-order</search:facet-option>
```

Comment:
Let's delete these 2 options files instead and then gitignore them. We don't want them to be version-controlled, because they'll be generated automatically as part of a deployment.

Reply:
Mentioned files deleted and already added src/main/entity-config to .gitignore as part of requirement.